README.md

```diff
@@ -1,6 +1,22 @@
-# Infratools utility repository
+# Heroku Buildpack: Populate `bash` environment from Vault
 
-## Introduction
+This buildpack populates `/etc/profile.d/` with a number of utility scripts to fetch
```

Comment:
To be exact, this places them in `$BUILD_DIR/.profile.d` (which ends up being `/app/.profile.d` in the runtime container)

Reply:
Thanks, this should be sorted now!